client/client.go

```diff
@@ -1139,6 +1140,22 @@ func (s *StarlightDaemonAPIServer) AddProxyProfile(ctx context.Context, req *pb.
 	}, nil
 }
 
+func (s *StarlightDaemonAPIServer) GetProxyProfiles(ctx context.Context, req *pb.Request) (resp *pb.GetProxyProfilesResponse, err error) {
+	log.G(s.client.ctx).WithFields(logrus.Fields{}).Debug("grpc: get proxy profiles")
```

Comment:
Just for housekeeping, maybe we don't need `.WithFields(logrus.Fields{})`here as we don't have anything fields to print

Reply:
I updated the commit! Thank you for feedback.